backend/common/src/main/java/ai/verta/modeldb/common/CommonConstants.java

```diff
@@ -13,6 +13,10 @@ public interface CommonConstants {
   String EMPTY_STRING = "";
   String S3 = "S3";
   Integer TAG_LENGTH = 40;
+  Integer THREAD_COUNT = 256;
+  String LIQUIBASE_MIGRATION = "LIQUIBASE_MIGRATION";
```

Comment:
recommend naming these env vars with something that indicates their usage as such. like `ENABLE_LIQUIBASE_MIGRATION_ENV_VAR` or something along those lines.

Reply:
Fixed